### PR TITLE
Add sites from issues

### PR DESCRIPTION
**Pull request by seia-soto**
refs #204, #146, #168, #161, #129, #157, #154, #128, #150, #145

### CODE REVIEW

File: db/organizations/cookieinformation.eno [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Cookie Information
+website_url: https://cookieinformation.com/
+privacy_policy_url: https://cookieinformation.com/cookie-and-privacy-policy/
+privacy_contact: privacy@cookieinformation.com
+country: DK
+description: "Offers your website visitors' transparency and give them control over what data is collected about them."
```
Comment (philipp-classen) [inline db/organizations/cookieinformation.eno:6]:
```suggestion
description: "Cookie Information is a Danish company that develops the consent management platform of the same name."
```

File: db/organizations/gainsight.eno [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: Gainsight
+website_url: https://www.gainsight.com/
+privacy_policy_url: https://www.gainsight.com/policy/privacy/
+privacy_contact: privacy@gainsight.com
+country: US
+description: Gainsight drives efficient growth by delivering customer interactions from end-to-end.
```
Comment (philipp-classen) [inline db/organizations/gainsight.eno:6]:
```suggestion
description: Gainsight operates in the SaaS sector, offering solutions for customer success and product experience. Their platform and applications support clients in making data-driven decisions regarding customer interactions.
```

File: db/organizations/conva.eno [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name: Conva Ventures
+website_url: https://usefathom.com/imprint
```
[the remark this answers is not in the document]
Comment (seia-soto) [reply db/organizations/conva.eno:2]: Oh, nice. I didn't notice there's an `/about` page. :+1:

**Review by philipp-classen — APPROVED** [review]

**Comment by seia-soto** [comment]
@philipp-classen The issues linked are not closed automatically. Please, take a look.

**Pull request merged**